readthedocs/proxito/middleware.py

```diff
@@ -184,7 +184,11 @@ def add_user_headers(self, request, response):
         with the ``Domain`` object.
         """
         if hasattr(request, 'domain'):
-            response_headers = [header.lower() for header in response]
+            # Use a private method to get this
+            # TODO: In Django 3.2 this has been upgraded to a top-level method
```

Comment:
"This" here is `response._headers.keys()`?

Reply:
Yep.